drivers/abp2/Kconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024 CNRS, France
```

Comment:
you can probably drop the Kconfig file we don't use Kconfig dependency modeling any more